src/service/sponsorService.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { ISponsor } from '../interfaces/ISponsor';
+
+const API_URL = 'http://localhost:7001';
+
+export const memberService = {
+  getAllMembers: async (): Promise<ISponsor[]> => {
+    const response = await axios.get(`${API_URL}/members`);
```

Comment:
we are just gonna use raw supabase bc it's faster, check out main

for now feel free to leave hard coded though, unless you want to add to supabase